packages/@aws-cdk-testing/framework-integ/test/aws-cognito/test/integ.user-pool-idp.oidc.ts

```diff
@@ -14,7 +14,7 @@ const userpool = new UserPool(stack, 'pool', {
 
 new UserPoolIdentityProviderOidc(stack, 'cdk', {
   userPool: userpool,
-  name: 'cdk',
+  name: 'my-provider',
```

Comment:
I think the pattern is right, but `cdk` is no longer working. Perhaps this is a new check from CloudFormation?

Reply:
This is a bit concerning, is this caused by the regex change?

Reply:
Have you tried `cdk` since the regex has been fixed? According to regex101, `cdk` should match.

Trying running the integration test again using the original name. 

If it works with the original name then great! We can keep the name `cdk` and simplify this PR.

If it fails with the original name, then determine if the failure is in CloudFormation or thrown by the CDK. If thrown by the CDK, see if running the same test from the `main` branch succeeds or fails in CloudFormation.

Reply:
Agreed, this was run before the update with the PCRE regex, cdh would not have worked before since it was checking for literal characters, p, Z etc, so if it didn't start with p/Z/_ it wouldn't have worked. But with the fix the start of the string should be saying that the first character cannot be a space or underscore.

Reply:
My bad, y'all. I ran it again but I had forgotten to build. We should be good now.